### ISSUE TITLE
Printer doesn't escape metadata strings

### ISSUE THREAD

**Issue by blais** [beancount/beancount#323]
**[Original report](https://bitbucket.org/blais/beancount/issue/323) by Christian Thiemann (Bitbucket: [c1thiemann](https://bitbucket.org/c1thiemann), GitHub: [cthiemann](https://github.com/cthiemann)).**

----------------------------------------

[beancount.parser.printer.EntryPrinter.write_metadata](https://bitbucket.org/blais/beancount/src/95ba37a428d29530fc3d565e14fd1386a6cd0bc5/beancount/parser/printer.py?at=default&fileviewer=file-view-default#printer.py-122,135,151,153) wraps string values in double quotes without escaping the value first, causing parsing errors when trying to load the generated file.

Possible fix (works for me locally): change line 135 to

```
#!python
    value_str = '"{}"'.format(misc_utils.escape_string(value))
```

**Comment by blais** [comment]
**Original comment by Jeremy Maitin-Shepard (Bitbucket: [jbms](https://bitbucket.org/jbms), GitHub: [jbms](https://github.com/jbms)).**

----------------------------------------

This problem is quite annoying for my own use cases, and I can confirm that this fix resolves the problem.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Fixed #323: Escape strings in printer.
